app/controllers/admin/application_controller.rb

```diff
@@ -34,7 +35,7 @@ def setup
     def order
       @order ||= Administrate::Order.new(
         params.fetch(resource_name, {}).fetch(:order, 'created_at'),
-        params.fetch(resource_name, {}).fetch(:direction, 'desc'),
+        params.fetch(resource_name, {}).fetch(:direction, 'desc')
```

Comment:
files in `admin` are mostly generated and should better not be formatted as it will make migrations harder

Reply:
👍